platform-enterprise_versioned_docs/version-25.1/enterprise/_templates/k8s/data_studios/proxy.yml

```
@@ -0,0 +1,73 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: connect-proxy
+  labels:
+    app.kubernetes.io/component: tower-connect-proxy
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/component: tower-connect-proxy
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/component: tower-connect-proxy
+    spec:
+      imagePullSecrets:
+        - name: "cr.seqera.io"
+      nodeSelector:
+        kubernetes.io/arch: amd64
+      containers:
+        - name: proxy
+          image: cr.seqera.io/private/nf-tower-enterprise/data-studio/connect-proxy:0.8
```

```suggestion
          image: cr.seqera.io/private/nf-tower-enterprise/data-studio/connect-proxy:0.8.0
```
Need to revert. My bad.